diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f46353b..f66d05df 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1237,10 +1237,12 @@ endif()
 if((KTX_FEATURE_TOOLS OR KTX_FEATURE_TESTS) AND NOT TARGET fmt::fmt)
     set(FMT_INSTALL OFF)
     set(FMT_SYSTEM_HEADERS ON)
-    add_subdirectory(external/fmt)
+    #add_subdirectory(external/fmt)
+    find_package(fmt CONFIG REQUIRED)
 endif()
 if(KTX_FEATURE_TOOLS AND NOT TARGET cxxopts::cxxopts)
-    add_subdirectory(external/cxxopts)
+    #add_subdirectory(external/cxxopts)
+    find_package(cxxopts CONFIG REQUIRED)
 endif()
 
 # Tools
