diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f46353b..b7121b93 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -520,7 +520,7 @@ macro(common_libktx_settings target enable_write library_type)
         CXX_STANDARD_REQUIRED YES
 
     )
-    if(APPLE_LOCKED_OS)
+    if(0)
         set_target_properties(${target} PROPERTIES
             FRAMEWORK TRUE
         )
@@ -1311,7 +1311,7 @@ endif()
 # Use of this to install KHR/khr_df.h is due to CMake's failure to
 # preserve the include source folder hierarchy.
 # See https://gitlab.kitware.com/cmake/cmake/-/issues/16739.
-if (APPLE_LOCKED_OS)
+if (0)
     set_source_files_properties(
           include/KHR/khr_df.h
           PROPERTIES MACOSX_PACKAGE_LOCATION Headers/KHR
